android/koin-android/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
 
 dependencies {
     api "io.insert-koin:koin-core:$koin_version"
+    api "androidx.navigation:navigation-runtime-ktx:$androidx_nav"
```

Comment:
navigation runtime can't be addded here. Another project is helping with it: koin-androidx-navigation.

Reply:
can we add navigation project dependency in compose or we create a new project with compose + navigation dependency?